app/service/user.py

```diff
@@ -0,0 +1,50 @@
+from fastapi import FastAPI
+from app.database import engine
+from sqlmodel import Session, select
+from app.models.models import User
+
+app = FastAPI()
+
+session = Session(bind=engine)
+
+
+async def read_users():
+    statement = select(User)
+    results = session.exec(statement).all()
+
+    return results
+
+
+async def read_user(user_id: int):
```

Comment:
```suggestion
async def read_user(user_id: int) -> User:
```
저희는 기본적으로 python typehint를 명시하는것으로 되어 있어서 이런식으로 return type도 명시해 주시면 좋을것 같아요

Reply:
typehint 명시하여 수정 했습니다. 감사합니다!